src/apispec/ext/marshmallow/schema_resolver.py

```diff
@@ -0,0 +1,218 @@
+from .common import resolve_schema_instance
+
+
+class SchemaResolver:
+    """Resolve Marshmallow Schemas in OpenAPI components and translate to OpenAPI
+    `schema objects
+    <https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#schema-object>`_,
```

Comment:
I think we've been linking to 3.0.2 in the rest of the docs.

```suggestion
    <https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#schema-object>`_,
```